src/OrangePaymentSlipData.php

```diff
@@ -54,6 +55,13 @@ class OrangePaymentSlipData extends PaymentSlipData
      * @var string
      */
     protected $bankingCustomerId = '';
+    
+    /**
+     * The banking customer ID expected length
+     *
+     * @var integer
+     */
+    protected $bankingCustomerIdLength = 0;
```

Comment:
I don't agree, if `$this->bankingCustomerId = '';`, `$this->bankingCustomerIdLength = 0;` must be set to `0` and not to `null`.

Futhermore, the specified type is integer, so it should never be null.